src/test/scala/thurloe/service/ThurloeServiceSpec.scala

```diff
@@ -27,7 +27,7 @@ class ThurloeServiceSpec extends FunSpec with ScalatestRouteTest {
 
   describe("The Thurloe Service") {
     it("should allow key/value pairs to be set") {
-      Post(uriPrefix, ukvp1) ~> thurloeService.routes ~> check {
+      Post(uriPrefix, ukvp1) ~> thurloeService.keyValuePairRoutes ~> check {
```

Comment:
Actually I'm not sure I changed anything here other than the name of the member from `routes` to `keyValuePairRoutes`; am I missing a more substantive change here?